collector/src/main/java/org/apache/hertzbeat/collector/collect/http/SslCertificateCollectImpl.java

```diff
@@ -159,7 +159,7 @@ private void validateParams(Metrics metrics) throws Exception {
         }
         HttpProtocol httpProtocol = metrics.getHttp();
         String url = httpProtocol.getUrl();
-        if (StringUtils.hasText(url) || !url.startsWith(RIGHT_DASH)) {
+        if (url == null || StringUtils.hasText(url) || !url.startsWith(RIGHT_DASH)) {
```

Comment:
I suggest that the same function:`StringUtils.isEmpty(url)` can be used for both of these string checks.